### PR TITLE
Removes dragoontomb ruin and loot.

### PR DESCRIPTION
**Pull request by AnturK**
It one big reference compounded with horrible quality sprites and no interesting function. The boomerang  spear might be spared maybe.

### CODE REVIEW

**Comment by Armhulen** [comment]
The boomerang spear is really buggy too, though. I think it would be better if you go all or nothing.

**Comment by Haevacht** [comment]
Name the bugs.
Do they have issue reports? Because I could have fixed them, potentially.
The multi-hitting thing was fixed, and noone told me it was a thing until rather late.

As is, this PR removes the lance too. So it'll fix whatever bug you've decided to withold from me.

**Comment by Armhulen** [comment]
If the multihitting bug was fixed then we're good.

**Comment by Penguaro** [comment]
There are times that Water Flan appear near the station, were they just migrating from this map or is there an event that calls them?

**Comment by Repukan** [comment]
They fly off the asteroid and drift until they hit something

**Comment by Haevacht** [comment]
That was not intentional.

File: code/modules/mob/living/simple_animal/hostile/flan.dm [hunk 1]
```diff
@@ -1,89 +0,0 @@
-//Will probably eventually be expanded to fit multiple types of Flan because I am a nerd.
-
-/mob/living/simple_animal/hostile/flan
```
Comment (Iamgoofball) [inline code/modules/mob/living/simple_animal/hostile/flan.dm]: can we keep these and put them on lavaland instead

Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/hostile/flan.dm]: no they suck it'd be better to give watchers their mechanics or something

**Comment by FantasticFwoosh** [comment]
If we are comparing scales of abuse, besides water flans floating into station space, i never see any abuse from the dragon tomb's loot compared to the syndicate grenade launcher, hardsuits & eswords from MMMiracles caravan attack map.

I don't think this needs to be removed without first asking the creator to make changes or concessions.

**Comment by Haevacht** [comment]
Let's make this more obivous yeah?
Hi, I made all this.

I want it gone.

File: code/modules/mob/living/simple_animal/hostile/flan.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/mob/living/simple_animal/hostile/flan.dm]: Why not just merge all mobs into one then

**Comment by dannno** [comment]
I never really liked that it got in to start with to be honest. Doesn't fit spacemen at all.

**Comment by Repukan** [comment]
Neither does almost 70% of the content in the server, yet here we are.

Stop using that as an excuse, it's shit and poor.

**Comment by Bawhoppen** [comment]
It's bad because it's just reused meme content, not because it's a reference

**Pull request merged**